client/src/components/DataFiles/DataFilesModals/DataFilesMkdirModal.js

```diff
@@ -66,7 +78,8 @@ const DataFilesMkdirModal = () => {
         >
           <Form>
             <ModalHeader toggle={toggle}>
-              Creating folder in: {params.path || '/'}
+              Creating folder in {systemDisplayName}
```

Comment:
For creating folders in existing folders in a system there is a problem with a missing `/`
![Screen Shot 2021-01-11 at 2 18 25 PM](https://user-images.githubusercontent.com/8287580/104244717-a34ba200-5428-11eb-939f-f41386f2f655.png)